### PR TITLE
Pull up 4 into master

### PR DESCRIPTION
**Pull request by AymericPlanche**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -301,7 +301,7 @@ workflows:
                 filters:
                     branches:
                         only:
-                            - master
+                            - "4.0"
```
Comment (jmleroux) [inline .circleci/config.yml]: I think it should still be master

File: src/Akeneo/Platform/CommunityVersion.php [hunk 2]
```diff
@@ -12,10 +12,10 @@
 class CommunityVersion
 {
     /** @staticvar string */
-    const VERSION = '3.2.18';
+    const VERSION = '4.0.0';
```
[the remark this answers is not in the document]
Comment (jmleroux) [reply src/Akeneo/Platform/CommunityVersion.php]: It's correct imho

[the remark this answers is not in the document]
Comment (ahocquard) [reply src/Akeneo/Platform/CommunityVersion.php]: We will change it later on if needed for serenity, not a problem for now.

**Comment by jmleroux** [comment]
Test frozen status

**Pull request merged**